cli/tests/test_git_clone.rs

```diff
@@ -159,6 +174,38 @@ fn test_git_clone() {
     "#);
 }
 
+#[test]
+fn test_git_clone_symlink() {
```

Comment:
CLI tests are *much* slower than lib tests. I think it should be sufficient to test this at the fileutil level in `file_util.rs`, or maybe at the working-copy level in `test_local_working_copy.rs`. Could you migrate this there?

Reply:
I think it makes sense to have a more end-to-end test than just `file_util`, since there's an implicit assumption about the format of the input arguments, so I'll look at moving it to a working-copy test.

Reply:
👍